astropy/coordinates/baseframe.py

```diff
@@ -508,9 +509,9 @@ def __init__(self, *args, copy=True, representation_type=None,
                     # come from the representation instead of the frame's
                     # attribute names.
                     try:
-                        representation_data = representation_cls._unit_representation(copy=copy,
-                                                                 **repr_kwargs)
-                    except Exception as e2:
+                        representation_data = representation_cls\
+                            ._unit_representation(copy=copy, **repr_kwargs)
+                    except Exception:
```

Comment:
does it have to be a bare Exception, or could be more specific?

Reply:
Changing the exception type is not in scope though, strictly speaking.

Reply:
yes, that's true. I've added the 🚲  for a reason 😉